src/main/java/com/google/step/snippet/external/GeeksForGeeksClient.java

```diff
@@ -6,12 +6,29 @@
 import org.jsoup.nodes.Document;
 import org.jsoup.select.Elements;
 
-public final class GeeksForGeeksClient {
+public final class GeeksForGeeksClient implements Client {
   private static final String TITLE_TAG = "h1";
   private static final String DESC_TAG = "p";
   private static final String SNIPPET_CLASS = "code-block";
   private static final String CODE_CLASS = "code-container";
 
+  private final String cseId;
+
+  public GeeksForGeeksClient(String cseId) {
+    this.cseId = cseId;
+  }
+
+  @Override
+  public String getCseId() {
+    return cseId;
+  }
+
+  /**
+   * Create and return a {@code Card} for the given W3School URL.
+   *
+   * @param geeksLink the URL of the W3School web page to create the card for.
```

Comment:
the biggest nit ever: according to Google JavaDoc style, there is no punctuation at the end of these fragments unless it is followed by another sentence. so remove the period, here and below